runelite-client/src/main/java/net/runelite/client/plugins/slayer/SlayerPlugin.java

```diff
@@ -571,22 +578,35 @@ private void addCounter()
 		}
 
 		Task task = Task.getTask(taskName);
-		int itemSpriteId = ItemID.ENCHANTED_GEM;
-		if (task != null)
-		{
-			itemSpriteId = task.getItemSpriteId();
-		}
+		int itemSpriteId = task == null ? ItemID.ENCHANTED_GEM : task.getItemSpriteId();
```

Comment:
Why was this changed to a ternary?

Reply:
Cleaner. It's not even needed anyways. `task` will never be null due to the check at the top of the function.

Reply:
I just thought it looked better. I simplified that entire function.

Reply:
I kind of agree here, that ternary looks nice.